dfastmi/gui/dialog_view.py

```diff
@@ -851,7 +857,9 @@ def _menu_load_configuration(self) -> None:
                     )
                 )
             else:
-                self._update_enabling_plotting_controls()
+                self._update_enabled_of_make_plot_dependent_view_items(
```

Comment:
this should be removed. The setting of the enabled flags should be done in the view model in load_config

Reply:
updated in pair programming